### PR TITLE
Replace [] with get() to get around missing fields

### PR DESCRIPTION
**Pull request by alixedi**
### Description of change

I found in dev that some companies from D&B did not have `registered_address` fields set. 

### Checklist

* [ ] Has a new newsfragment been created? Check [changelog/README.md](https://github.com/uktrade/data-hub-leeloo/blob/master/changelog/README.md) for instructions
* [ ] Do any added or updated endpoints appear in the API documentation? See [docs/Maintaining the API documentation.md](https://github.com/uktrade/data-hub-leeloo/blob/develop/docs/Maintaining&#32;the&#32;API&#32;documentation.md) for more details
* [ ] Have any relevant search models been updated?
* [ ] Have any relevant fixtures (`fixtures/test_data.yaml`) been updated?
* [ ] Have any relevant select-/prefetch-related field lists in the views and search apps been updated?
* [ ] Has the admin site been updated (for new models, fields etc.)?
* [ ] Has the README been updated (if needed)?

### CODE REVIEW

File: datahub/company/templates/admin/company/company/update-from-dnb.html [hunk 1]
```diff
@@ -30,7 +30,7 @@
       </thead>
       {% for field, value in diff.items %}
       <tr>
-        <td>{{field.verbose_name|capfirst}}</td>
+        <td>{{field.verbose_name|title}}</td>
```
Comment (reupen) [inline datahub/company/templates/admin/company/company/update-from-dnb.html]: `capfirst` is what Django uses – also doesn't seem to be related to this change?

Comment (alixedi) [reply datahub/company/templates/admin/company/company/update-from-dnb.html]: Looked better with title so I thought I'd sneak it in :D
